packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -396,6 +397,17 @@ class ApplicationCommand extends Base {
     }
     return true;
   }
+  
+  /**
+   * When concatenated with a string, this automatically returns the command's mention instead of the ApplicationCommand object.
+   * @returns {string}
+   * @example
+   * // Logs: Command: </ping:123456789012345678>
+   * console.log(`Command: ${command}`);
+   */
+  toString() {
+    return chatInputApplicationCommandMention(this.name, this.id);
```

Comment:
```suggestion
    return this.type === ApplicationCommandType.ChatInput
      ? chatInputApplicationCommandMention(this.name, this.id)
      : '[object Object]';
```
Shouldn't it check the type?

Reply:
Did you forgot to change the return type in d.ts?

Reply:
Ah, right.  Would you suggest doing `ChatInputApplicationCommandMention | string`, or just `string`?  The former would make it more consistent with types that return UserMention, RoleMention, etc. although those can *only* return those types.